src/Sulu/Bundle/ContactBundle/Contact/ContactManager.php

```diff
@@ -257,7 +269,7 @@ public function save(
                 $parent = $this->accountRepository->findAccountById($parentData['id']);
                 if (!$parent) {
                     throw new EntityNotFoundException(
-                        $this->getAccountEntityName(),
+                        self::$accountContactEntityName,
```

Comment:
@danrot getAccountEntityName did not exists. Apparently the method was removed a while ago and the changer forgot to refactor this line of code